packages/tatum-moonbeam/src/multiToken/multiToken.spec.ts

```diff
@@ -0,0 +1,127 @@
+import {
+  burnMultiToken,
+  burnMultiTokenBatch,
+  deployMultiToken,
+  mintMultiToken,
+  mintMultiTokenBatch,
+  transferMultiToken,
+  transferMultiTokenBatch,
+} from './index'
+
+describe('NFT tests', () => {
+  jest.setTimeout(99999)
+
+  describe('NFT moonbeam 1155 transactions', () => {
+    it('should test moonbeam 1155 deploy transaction', async () => {
+      const deployMultiTokenToken = await deployMultiToken(
+        {
+          fromPrivateKey: '0x1a4344e55c562db08700dd32e52e62e7c40b1ef5e27c6ddd969de9891a899b29',
+          uri: 'tatum',
+        },
+        'https://rpc-mumbai.matic.today'
```

Comment:
Provider in spec files has been fixed. But It is not testable now, because of moonbeam alpha (testnet) problem. But I did integration test this week by REST client and swagger tool as well.